src/components/application_manager/test/hmi_capabilities_test.cc

```diff
@@ -1124,6 +1124,35 @@ TEST_F(HMICapabilitiesTest,
   EXPECT_FALSE(file_system::FileExists(kHmiCapabilitiesCacheFile));
 }
 
+TEST_F(
+    HMICapabilitiesTest,
+    SameCcpuVersionReceived_NoRequiredRequestsForCapability_RespondAllRAIRequests) {
+  MockApplicationManagerSettings mock_application_manager_settings;
+  const std::string ccpu_version = "4.1.3.B_EB355B";
+
+  hmi_capabilities_->set_ccpu_version(ccpu_version);
+
+  EXPECT_EQ(ccpu_version, hmi_capabilities_->ccpu_version());
+  EXPECT_CALL(mock_app_mngr_, SetHMICooperating(true));
+
+  hmi_capabilities_->OnSoftwareVersionReceived(ccpu_version);
+}
+
+TEST_F(
+    HMICapabilitiesTest,
+    SameCcpuVersionReceived_RequestsAreRequiredForCapabilities_NotRespondAllRAIRequests) {
```

Comment:
@ydementieiev NotRespondAllRAIRequests - I think it's not obvious here, maybe add some comments why in the first test you do not expect any capabilities requests, but here you expect them

Reply:
@LitvinenkoIra Thank you, I added some description, you can check